NuRadioMC/test/SingleEvents/T03validate.py

```diff
@@ -82,7 +82,7 @@
         print(e)
         error = -1
 
-keys2 = [u'SNRs',
```

Comment:
Why don't we test the SNRs anymore?

Reply:
I removed SNRs from the output file as this is just a duplication of data. The same information is contained in amplitude. Just divide the amplitude by the fixed RMS noise and one gets the SNR.